frontend/src/components/FilterMenu.js

```diff
@@ -62,8 +79,14 @@ export function FilterItem({ filter, onRemoveFilter, onUpdateFilter }) {
     }
   });
 
-  const onUpdate = (name, value) => {
-    onUpdateFilter(id, name, value);
+  const onUpdate = (name, value, toggleAllChecked = true) => {
+    if (!query && name === 'condition') {
+      // Set default value.
+      const defaultQuery = DEFAULT_VALUES[topic][value];
+      onUpdateFilter(id, 'query', defaultQuery, toggleAllChecked);
```

Comment:
I don't think this component should care about the content of the filters, is there anyway we can remove `toggleAllChecked` and figure that out in the pill?

After looking it seems like the `CheckboxSelect` tracks if the `all` checkbox is checked. I wonder if we can dynamically determine the value of `all checked` by looking if all options have been selected or not. Might be nice to have the `all` checkbox selected if a user individually selects each option.

Reply:
As discussed, this was at the request of Kelly to get a similar behavior to the filter on other pages.

Reply:
After talking about this with @AdamAdHocTeam (correct me if I get this wrong!)... We want to be able to distinguish in the UI when a user selects the "all" option in a multi-select vs when they select all options individually. This seems fine for now, we may come up with a slick (and understandable!) way of handling it in the future.